src/test/java/io/rebloom/client/ClusterClientTest.java

```diff
@@ -185,15 +183,32 @@ public void addExistsMulti() {
 
         rv = ccl.addMulti("newFilter", new byte[]{0}, new byte[]{3});
         assertEquals(2, rv.length);
-        TestCase.assertTrue(rv[0]);
-        TestCase.assertFalse(rv[1]);
+        assertTrue(rv[0]);
+        assertFalse(rv[1]);
 
         rv = ccl.existsMulti("newFilter", new byte[]{0}, new byte[]{1}, new byte[]{2}, new byte[]{3}, new byte[]{5});
         assertEquals(5, rv.length);
-        TestCase.assertTrue(rv[0]);
-        TestCase.assertTrue(rv[1]);
-        TestCase.assertTrue(rv[2]);
-        TestCase.assertTrue(rv[3]);
-        TestCase.assertFalse(rv[4]);
+        assertTrue(rv[0]);
+        assertTrue(rv[1]);
+        assertTrue(rv[2]);
+        assertTrue(rv[3]);
+        assertFalse(rv[4]);
+    }
+
+    @Test
+    public void createTopKFilter() {
+        ccl.topkCreateFilter("aaa", 30, 2000, 7, 0.925);
```

Comment:
We should extend this test with more than a single key, to verify the cluster behavior

Reply:
do it.